src/main/webapp/app/admin/privacy-statement/unsaved-changes-warning/privacy-statement-unsaved-changes-warning.component.html

```diff
@@ -0,0 +1,12 @@
+<form>
+    <div class="modal-header">
+        <button id="close-modal-btn" type="button" class="btn-close" data-dismiss="modal" aria-hidden="true" (click)="continueEditing()"></button>
+    </div>
+    <div class="modal-body">
+        <h5 jhiTranslate="artemisApp.privacyStatement.unsavedChangesWarning"></h5>
+        <div>
+            <jhi-button id="discard-content-btn" (click)="discardContent()" [title]="'global.generic.yes'" [ngStyle]="{ 'marginRight.px': 10 }">Yes</jhi-button>
```

Comment:
```suggestion
            <jhi-button id="discard-content-btn" (click)="discardContent()" [title]="'global.generic.yes'">Yes</jhi-button>
```
non-dynamic style should probably be part of the scss